app/aem/src/server/aem-component-loader.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path');
+const parser = require('xml2json');
+const txtLoader = require.resolve('./aem-clientlib-txt-loader.js');
+const JCR_ROOT_KEY = 'jcr:root';
+const JCR_TITLE_KEY = 'jcr:title';
+
+const getRequiredHTL = (component, context, pathBaseName) => {
+  return `const component = ${JSON.stringify(component)};
+   component.module = require('${context}/${pathBaseName}.html');
+   module.exports = component;`;
+}
```

Comment:
Not sure if we need this at all - but if we do we should generate only ES5 so that we can support IE11:

```suggestion
const getRequiredHTL = (component, context, pathBaseName) => {
  return `var component = ${JSON.stringify(component)};
   component.module = require('${context}/${pathBaseName}.html');
   module.exports = component;`;
}
```

Also from this code it is really hard to understand what `component` is